ompi/mca/mtl/ofi/mtl_ofi_component.c

```diff
@@ -392,7 +410,7 @@ ompi_mtl_ofi_component_init(bool enable_progress_threads,
     if (FI_ENODATA == -ret) {
         // It is not an error if no information is returned.
         goto error;
-    } else if (0 != ret) {
+    } else if (OPAL_UNLIKELY(0 != ret)) {
```

Comment:
Remove the compiler hint; this is initialization, not the critical path.

Reply:
Ok, will remove all compiler hints from init time.